dpypx/autodraw.py

```diff
@@ -28,7 +28,9 @@ def load_image(
             image = new_image
         width = round(image.width * scale)
         height = round(image.height * scale)
-        resized = image.resize((width, height), Image.BILINEAR)
+        # since the image is loaded once
+        # we can afford to use a higher quality filter than bilinear
```

Comment:
```suggestion
        # Since the image is only loaded once we can afford to use a high
        # quality filter.
```
Capitalisation, consistent line breaks, not making a reference to code that no longer exists.